aspnetcore/fundamentals/native-aot.md

```diff
@@ -5,20 +5,37 @@ description: Learn about ASP.NET Core support for native AOT
 monikerRange: '>= aspnetcore-8.0'
 ms.author: midenn
 ms.custom: mvc
-ms.date: 04/10/2023
+ms.date: 4/10/2023
 uid: fundamentals/native-aot
 ---
 # ASP.NET Core support for native AOT
 
-ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). This article covers:
-
-* How to get started.
-* The benefits and potential drawbacks.
-* Compatibility requirements.
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/).
 
 > [!WARNING]
 > In .NET 8, not all ASP.NET Core features are compatible with native AOT.
 
+## Prerequisites
+
+# [.NET Core CLI](#tab/netcore-cli) 
+
+* [!INCLUDE[](~/includes/8.0-SDK.md)]
+* From the Visual Studio installer, add the Desktop development with C++
```

Comment:
@mitchdenny This is not a good CLI experience. Is that going to be fixed soon? You need to install the VS installer, then install a VS component to run a CLI command?

Reply:
@mitchdenny @Rick-Anderson It looks like this question didn't get addressed. Shouldn't the prerequisites refer to **dotnet workload install** CLI commands rather than VS installation? If you have to install Visual Studio that makes this a Windows-only procedure, but nowhere in the article is that made explicit.